packages/elements/src/datetime-picker/index.ts

```diff
@@ -1088,6 +1093,46 @@ export class DatetimePicker extends FormFieldElement implements MultiValue {
     this.validateInput();
   }
 
+  /**
+   * Run when tap event happens on clears button and fire value-changed event.
+   * @param event tap event
+   * @returns {void}
+   */
+  protected override onClearsButtonTap(event: TapEvent): void {
+    let values: string[] = [];
+    if (this.inputFromEl && this.inputToEl) {
+      // range mode
+      this.inputFromEl.value = '';
+      this.inputToEl.value = '';
+      // When the element's input is cleared by typing,
```

Comment:
Let's move this comment block before the conditional statement (above line 1103). It applies to both range and single mode.

Reply:
Fixed.